### ISSUE TITLE
Make MATE panels more accessible with Orca screen reader in Mint MATE 22

### ISSUE THREAD

**Issue by slannon97** [linuxmint/linuxmint#693]
When I press Control Alt Tab to switch to the bottom panel in Mint 22 MATE, Orca doesn't read all the tray icons to me when I tab or arrow around with the arrow keys. It will read the clock, but many other items such as the icon for Wi-Fi or system updates do not read at all. Instead, Orca says panel or panel notification area panel. If this should be reported to MATE directly, please let me know, but I figured I'd report it here, as Mint seems to be using a customized version of MATE. Please fix it so Orca reads what icon is selected, and allow keyboard focus to activate associated menus and options by pressing the space bar or enter. As it stands right now, I'm unable to access some features from the GUI such as network status because of this issue. This issue is very similar to the prior one I posted about the inaccessible Update and Software Manager applications. If the fix isn't immediate, I hope it will at least arrive with Mint 22.1 in December. I want Linux Mint to be as accessible as possible for blind users that rely on Orca.

**Comment by billyswong** [comment]
You really should file this issue to https://github.com/mate-desktop/mate-panel/issues, together with a link back here, unless you find the same bug disappears if you run Ubuntu MATE 24.04.

**Comment by slannon97** [comment]
The same issues exist with Ubuntu MATE as well. On a version of Arch running MATE 1.28.2 for my Raspberry Pi 4, Orca seems to read all icons, including network, Bluetooth, and volume just fine. I'm not sure I should report anything until Mint is updated to MATE 1.28.2. Is this planned at some point in the 22 cycle, or will I have to wait until Mint 23?

**Comment by billyswong** [comment]
Linux Mint upgraded from MATE 1.24 to MATE 1.26 in LM 20.3, in step with Ubuntu MATE 21.10. So it is definitely possible to happen within the 22 cycle.

You may also file the issue to https://github.com/linuxmint/xapp/issues for the XApp Status Applet, which is where many tray icons (but not all) in Linux Mint MATE edition living in. This part is developed by Linux Mint and not by MATE desktop.

**Comment by slannon97** [comment]
This issue is still present in Mint 22.1. I created an issue as suggested in the previous comment, and here's the link.
https://github.com/linuxmint/xapp/issues/188
